src/rgw/rgw_pubsub.h

```diff
@@ -593,10 +593,70 @@ class RGWPubSub
     int remove_notifications(const DoutPrefixProvider *dpp, optional_yield y) const;
   };
 
+<<<<<<< HEAD
```

Comment:
It seems like a rebase went wrong here.  Conflict markers obviously don't compile ;)

Reply:
Sigh 🤦 . The number of conflicts was really high. If I only managed one (and I was doing this late at night), I'll take that as a win. I'll of course fix it.